packages/core/src/browser/preferences/preference-contribution.ts

```diff
@@ -188,13 +188,19 @@ export class PreferenceSchemaProvider extends PreferenceProvider {
         return inverseChanges;
     }
 
-    protected doSetSchema(schema: PreferenceSchema): PreferenceProviderDataChange[] {
+    private validateSchema(schema: PreferenceSchema): void {
```

Comment:
```suggestion
    protected validateSchema(schema: PreferenceSchema): void {
```

Reply:
I would keep it private. This PR does not want to allow custom preference schema validation.

Reply:
> I would keep it private. This PR does not want to allow custom preference schema validation.

Do you have a compelling reason why the framework should _not_ allow custom preference schema validation, if for example a downstream project prefers a different library to AJV or wants to implement a custom check?